drafts/event-storming.md

```diff
@@ -0,0 +1,78 @@
+---
+title: DDD: How to do Event-Storming?
+domain: campzulu.hashnode.dev
+tags: software-architecture, ddd
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1670243050541/SwGJDMFY4.png?auto=compress
+publishAs: keerthikan
+ignorePost: true
+---
+
+# DDD: How to do Event Storming?
+
+At Zühlke, there is a **DDD Topic Team** to discuss about various topics related to Domain-Driven-Design. Recently, we had a session about **Event Storming**. In this post, I would like to share how we did it and some key takeaways.
```

Comment:
I believe we shoudl use the English wording `Zuehlke`.

Reply:
I think we should use `Zühlke` because that's the company name.
Besides I see `Zühlke` in our English web pages as well (e.g. https://www.zuehlke.com/en)